lib/excon/constants.rb

```diff
@@ -119,10 +119,10 @@ module WaitWritable; end
     :idempotent           => false,
     :instrumentor_name    => 'excon',
     :middlewares          => [
+      Excon::Middleware::Instrumentor,
```

Comment:
Yeah, I think we have run into this before. Not sure what the best approach would be.